website/components/live-example/LiveExample.tsx

```diff
@@ -146,7 +154,15 @@ function LiveExample<ComponentProps extends ComponentPropsInterface>({
           );
 
         case Knob.Number:
-          return <Number {...sharedProps} value={props[propName] as number} />;
+          return (
+            <Number
+              {...sharedProps}
+              value={props[propName] as number}
+              min={knobConfig?.min}
+              max={knobConfig?.max}
```

Comment:
Typescript not believing that min, max and step are valid values, but They've been added to the KnobConfigInterface (lines 94-99)

Reply:
@lazytype any thoguhts on what I'm doing wrong here?

Reply:
Most of the `*ConfigInterface` types don't have `min`, `max`, and `step` properties, so attempting to `Extract` the properties from them wouldn't have any effect. This `switch` statement also doesn't have any narrowing effect, as you can tell from the need to assert `knobConfig?.options as Array<string>` below in the `Knob.Select` case.

Reply:
👍